src/CommandLineUtils/Abstractions/DefaultValueParserFactory.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace McMaster.Extensions.CommandLineUtils.Abstractions
+{
+    using System;
```

Comment:
Nit: the formatting of your code doesn't exactly match with the rest of the project, but I don't blame you -- Visual Studio (and .NET in general) don't provide me with good ways to help new contributors follow a consistent code format. After this is merged, I'll reformat to fit with the rest of the project. That will change a few things, such as moving usings to the top of the file, adding accessibility modifiers, removing regions, and adding braces to conditional code. Of course, you're welcome to make those changes yourself and push and update to this PR, but it's not a requirement :)

Reply:
Can Roslyn based extensions help with enforcing the chosen code style?

Reply:
Know of any good ones? My criteria is that I need a tool that works on my macbook and can be easily used by all contributors without installing something heavy like ReSharper. I've toyed with integrating `dotnet-format` into the build.ps1 script, but it's not very good yet and always gives me false warnings on my macbook due to it's broken implementation of loading the MSBuild project system.

Reply:
Not really. But it is something I am considering to look into, when the time comes.